src/app/view/components/dot-navigation/dot-navigation.service.ts

```diff
@@ -40,7 +40,10 @@ export class DotNavigationService {
 
         this.loginService.auth$
             .filter((auth: Auth) => !!(auth.loginAsUser || auth.user))
-            .mergeMap((auth: Auth) => this.reloadNavigation().filter((isPortletInMenu: boolean) => !isPortletInMenu))
+            .mergeMap((auth: Auth) =>
```

Comment:
Found a bug that when you login, logout and then try to redirect to previousSavedURL it wasn't working, so I fix it here.

I didn't do the same thing for the edit redirect because if you to to /edit it will reload the angular app again.